src/katgpucbf/__init__.py

```diff
@@ -42,6 +42,7 @@
 DEFAULT_JONES_PER_BATCH: Final = 2**20
 DIG_HEAP_SAMPLES: Final = 4096
 DIG_SAMPLE_BITS: Final = 10
+ENGINE_DITHER_SEED_BITWIDTH: Final = 2**64
```

Comment:
The value isn't a BITWIDTH - the bitwidth is 64. I'd suggest either
- Change the value to 64, and use `2**ENGINE_DITHER_SEED_BITWIDTH`; or
- Change this to `ENGINE_DITHER_SEED_MAX: Final = 2**64 - 1`, then use `randint` instead of `randrange`.

Reply:
Argh, you are too right! My apologies. I must've been on autopilot with the naming. I might go with the former to follow even just the line above.